core/control/src/ext-event/ExtEventController.ts

```diff
@@ -0,0 +1,76 @@
+import 'reflect-metadata';
+import * as express from 'express';
+import { inject, injectable } from 'inversify';
+import {
+  controller,
+  httpGet,
+  httpPost,
+  interfaces,
+} from 'inversify-express-utils';
+import {
+  ApiOperationGet,
+  ApiOperationPost,
+  ApiPath,
+  SwaggerDefinitionConstant,
+} from 'swagger-express-ts';
+import { ExtEventService } from './ExtEventService';
+import { ExtEventRequestModel } from './ExtEventRequestModel'
+import { MessageService } from '../svc/MessageService';
+import { WebSocketService } from '../svc/WebSocketService';
+
+const uuidv4 = require('uuid/v4');
+
+@ApiPath({
+  name: 'Event',
+  path: '/event',
+  security: { apiKeyHeader: [] },
+})
+@controller('/event')
+export class ExtEventController implements interfaces.Controller {
+
+  constructor(@inject('ExtEventService') private readonly extEventService: ExtEventService) {}
+
+  @ApiOperationPost({
+    description: 'Handle incoming external event',
+    parameters: {
+      body: {
+        description: 'Keptn CloudEvent',
+        model: 'ExtEventRequestModel',
+        required: true,
+      },
+    },
+    responses: {
+      200: {
+      },
+      400: { description: 'Parameters fail' },
+    },
+    summary: '',
```

Comment:
Please provide a meaningful summary and explanation of the error codes. We should also do that for all other swagger-ized components.

Reply:
Thank you for pointing out. I added a meaningful summary.
I agree that we have to work on the error codes and their description. Now, we use in all endpoints the same description (i.e. _Parameters fail_) as it is used in the example of the library (i.e. https://github.com/olivierlsc/swagger-express-ts).